apps/ui/src/networks/offchain/api/index.ts

```diff
@@ -83,11 +83,17 @@ function getProposalState(
   if (proposal.state === 'closed') {
     const currentQuorum = getProposalCurrentQuorum(networkId, {
       scores: proposal.scores,
-      scores_total: proposal.scores_total
+      scores_total: proposal.scores_total,
+      quorum_type: proposal.quorumType
     });
 
-    if (currentQuorum < proposal.quorum) return 'rejected';
-
+    if (
+      (proposal.quorumType === 'rejection' &&
+        currentQuorum > proposal.quorum) ||
+      (proposal.quorumType !== 'rejection' && currentQuorum < proposal.quorum)
+    ) {
+      return 'rejected';
+    }
```

Comment:
```suggestion
    if (
      proposal.quorumType === 'rejection'
        ? currentQuorum > proposal.quorum
        : currentQuorum < proposal.quorum
    ) {
      return 'rejected';
    }

```

Would write it like this as we are checking `quorumType` against `"rejection"` twice in the same condition.

Other than that it looks good to me.